### PR TITLE
Sponsored popup abstract

### PR DESCRIPTION
**Pull request by uplne**
Created fragment for paid for content meta block so we don't need to edit it in multiple places.
- I did the requested change to replace 'created' with 'produced'

screen shot 2016-01-29 at 13 56 03

CC @kelvin-chappell @jbreckmckye @Calanthe

### CODE REVIEW

File: common/app/views/fragments/commercial/paidForMeta.scala.html [hunk 1]
```diff
@@ -0,0 +1,14 @@
+@()(implicit request: RequestHeader)
```
Comment (kelvin-chappell) [inline common/app/views/fragments/commercial/paidForMeta.scala.html]: Is the `request` needed here?

**Comment by kelvin-chappell** [comment]
Apart from comment :+1:

File: common/app/views/fragments/commercial/paidForMeta.scala.html (hunk 1 above)
Comment (uplne) [inline common/app/views/fragments/commercial/paidForMeta.scala.html]: LInkTo required it

Comment (kelvin-chappell) [inline common/app/views/fragments/commercial/paidForMeta.scala.html]: Then :ship:

**Pull request merged**